Source/Mosa.Compiler.x64/Intrinsic/SetCR2.cs

```diff
@@ -14,8 +14,8 @@ private static void SetCR2(Context context, Transform transform)
 	{
 		Operand operand1 = context.Operand1;
 
-		Operand eax = Operand.CreateCPURegister32(CPURegister.RAX);
```

Comment:
need to use 64 bit registers, should use Allocate64 not Allocate32
should rename eax to rax